### PR TITLE
fix: Allow more email addresses in notification lists

### LINKED ISSUE #4736: Maximum number of email addresses on send notices to
### Describe the issue

I cannot add 16 email addresses to the send notices.  Is it possible to increase the limit?  Is the editing limit due to characters or number of email addresses. 

Background.  Draft was limited to 2 editors from a combination of authors + editors.   The editors want to automatically notify the original authors and editors. 

page to reference draft-ietf-idr-bgp-car-00.txt 

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by larseggert**
Fixes #4736

### CODE REVIEW

**Review by rjsparks — CHANGES_REQUESTED** [review]
This is insufficient - a matching change to the document model is also needed, or the extra bits won't be stored in the database. See  https://github.com/ietf-tools/datatracker/blob/main/ietf/doc/models.py#L111. 

It's not immediately clear whether leaving this as a `CharField` in the model and making it significantly larger is the right thing to do. I suspect it probably needs to be converted to a `TextField`. Either way, the change will need to be accompanied by a migration effecting the change in the database.

(You build that with something like `ietf/maange.py makemigrations doc` after changing the model.)

File: ietf/doc/migrations/send_notices.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.28 on 2022-12-05 14:37
```
Comment (rjsparks) [inline ietf/doc/migrations/send_notices.py:1]: This needs to be renamed to be 0048_send_notices.py (Though i'd suggest 0048_longer_notify_field.py)

**Review by rjsparks — CHANGES_REQUESTED** [review]

**Review by rjsparks — APPROVED** [review]

**Review by jennifer-richards — APPROVED** [review]
Only comment is that I'd probably have named the migration something more descriptive (`0048_lengthen_notification_lists` or something), but that's a small nit. Looks good otherwise.

**Comment by uzkjzjjz** [comment]
Notes_221114_232000_f65

**Pull request merged**